.circleci/config.yml

```diff
@@ -103,6 +103,10 @@ commands:
           name: start memcached
           command: |
             docker run --rm -d -p 11211:11211 memcached -m 64
+      - run:
+          name: start redis
+          command: |
+            docker run --rm -d -p 6379:6379 redis -m 64
```

Comment:
I think CI will pass if you remove the `-m 64` from the end of this line.

Reply:
I'm very sorry it's taken so long to deal with this issue, I'll fix it and resubmit the PR